components/de_amp/common/features.cc

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/components/de_amp/common/features.h"
+
+#include "base/feature_list.h"
+
+namespace de_amp {
+namespace features {
+
+// When enabled, Brave will try to parse tracking URLs of the form
+// https://tracker.example.com/?url=http://final.example.com/
+// and automatically redirect to the final URL without sending any
+// network requests to the tracker.
```

Comment:
how it's different from this? https://github.com/brave/brave-core/pull/9065
seems like the same thing.

Reply:
apologies, that was an old comment, I'll delete that. That is not related to this PR, but is related to a different De-AMPing feature.

Reply:
This PR is for 1 in https://docs.google.com/document/d/1kvHbQMdE_vxmRWRz0LfAHzf6XoTT_pQXSwxth5rVhsk/edit#heading=h.fybr2a4q8y2c and that comment is referring to 3 https://docs.google.com/document/d/1kvHbQMdE_vxmRWRz0LfAHzf6XoTT_pQXSwxth5rVhsk/edit#heading=h.pl4yanc4il62

Reply:
gotcha, thanks!